pypit/_version.py

```diff
@@ -0,0 +1,2 @@
+__version__ = '0.6.dev0'
```

Comment:
To be honest, I copied the DESI approach.  Not sure why
they named it this way.  

According to the great Google, this is the preferred way:

http://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package